src/alcotest-engine/pp.ml

```diff
@@ -212,7 +212,7 @@ struct
       | 0 -> green_s ppf "Test Successful"
       | n -> red ppf "%d failure%a!" n pp_plural n
     in
-    Fmt.pf ppf "%a in %.3fs. %d test%a run.@," pp_failures r.failures r.time
+    Fmt.pf ppf "%a in %.3fs. %d test%a run.@\n" pp_failures r.failures r.time
```

Comment:
This is where we have my controversial change about a new line which does not appear anymore. Not sure if it's a story about `Format`'s boxes or the default margin of a `Format.formatter`.